packages/toolkit/src/view/catalog/components/notifications/CatalogLimitNotification.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, Dialog, Icons } from "@instill-ai/design-system";
+
+export type CatalogLimitNotificationProps = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+export const CatalogLimitNotification = ({
+  isOpen,
+  onClose,
+}: CatalogLimitNotificationProps) => {
+  return (
+    <Dialog.Root open={isOpen} onOpenChange={onClose}>
```

Comment:
@thewbuk This naming is not straight forward, since onOpenChange will get triggered when dialog is opened.

You can directly name it as `setIsOpen` or `onOpenChange`